components/ActionSheet/src/private/MDCActionSheetItemTableViewCell.m

```diff
@@ -155,7 +158,18 @@ - (void)updateTitleFont {
   [self setNeedsLayout];
 }
 
+- (void)setAdjustsFontForContentSizeCategory:(BOOL)adjustsFontForContentSizeCategory {
+  self.actionLabel.adjustsFontForContentSizeCategory = adjustsFontForContentSizeCategory;
+}
+
+- (BOOL)adjustsFontForContentSizeCategory {
+  return self.actionLabel.adjustsFontForContentSizeCategory;
+}
+
 - (void)mdc_setAdjustsFontForContentSizeCategory:(BOOL)adjusts {
+  if (_mdc_adjustsFontForContentSizeCategory == adjusts) {
```

Comment:
Is this required for the UIContentSizeCategoryAdjusting behavior or can it be split to another PR?